specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2025-04-01/containerregistry.json

```diff
@@ -6047,6 +6017,7 @@
       "in": "query",
       "name": "$filter",
       "description": "An OData filter expression that describes a subset of connectedRegistries to return. The parameters that can be filtered are parent.id (the resource id of the connectedRegistry parent), mode, and connectionState. The supported operator is eq.",
+      "required": false,
```

Comment:
Is it auto-generated? I don't see other places have `required` property value as `false`.

Reply:
Same as other comments. It was copied from the previous swagger. Dont think it is required technically. But I will leave it here to keep it in sync